pom.xml

```diff
@@ -40,16 +40,29 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.289.1</jenkins.version>
-        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+        <changelist>
+            999999-SNAPSHOT</changelist>
+        <jenkins.version>2.361.4</jenkins.version>
+        <gitHubRepo>
+            jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     </properties>
+    <build>
+        <extensions>
+          <extension>
+            <groupId>kr.motd.maven</groupId>
+            <artifactId>os-maven-plugin</artifactId>
+            <version>1.6.2</version>
+            </extension>
+        </extensions>
+    <plugins>
+        </plugins>
+    </build>
```

Comment:
You need to remove this still, but like I mentioned it before, the build will continue to fail until a maintainer replays it.

Reply:
Thank you i was very confused about why does it keep failing. I'll let this PR be until a maintainer replays it

Reply:
> until a maintainer replays it

FYI https://github.com/jenkinsci/workflow-multibranch-plugin/pull/220